component/grant-type/src/main/java/org/wso2/carbon/identity/oauth2/grant/jwt/JWTBearerGrantHandler.java

```diff
@@ -483,7 +483,7 @@ private boolean checkNotBeforeTime(Date notBeforeTime, long currentTimeInMillis,
      */
     private boolean checkValidityOfTheToken(Date issuedAtTime, long currentTimeInMillis, long timeStampSkewMillis) throws IdentityOAuth2Exception {
         long issuedAtTimeMillis = issuedAtTime.getTime();
-        long rejectBeforeMillis = validityPeriod * 60 * 1000;
+        long rejectBeforeMillis = 1000L * 60 * validityPeriod;
```

Comment:
check this whether L can be put in outside.
will it not affect the calculation? no truncation errors.